src/app/lib/analyticsUtils/index.js

```diff
@@ -11,7 +11,23 @@ import {
   SUPPORTED_MEDIUM_CAMPAIGN_TYPES,
 } from './analytics.const';
 
-export const getDestination = statsDestination => {
+export const getAmpDestination = (psDestination, gnlDestination) => {
+  /*
+   ** AMP Variable substitution is used here to construct a string that
+   ** will get the correct destination according to the client's geolocation,
+   ** found using <amp-geo>.
+   ** https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md
+   ** https://amp.dev/documentation/components/amp-geo/
+   */
+
+  // eslint-disable-next-line no-template-curly-in-string
+  const ampGeo = '${ampGeo}'; // String representing the list of matched groups (comma delimited) e.g. eea,gbOrUnknown
+  const withGbOrUnknownMatched = `$MATCH(${ampGeo}, gbOrUnknown, 0)`; // Checks for presence of 'gbOrUnknown' and returns 'gbOrUnknown' if found
+  const equalsgbOrUnknown = `$EQUALS(${withGbOrUnknownMatched}, gbOrUnknown)`; // Returns 'true' if the result of the $MATCH was 'gbOrUnknown'
```

Comment:
Can we follow the common `isNaming` convention for boolean variables here? For example, instead of "equals" this could be `isGbOrUnknown`.